docs/backtesting.md

```diff
@@ -215,22 +179,33 @@ time a trade will reach 1%).
 ```
 
 On the other hand, if you set a too high `minimal_roi` like `"0":  0.55`
-(55%), there is a lot of chance that the bot will never reach this
-profit. Hence, keep in mind that your performance is a mix of your
-strategies, your configuration, and the crypto-currency you have set up.
+(55%), there is almost no chance that the bot will never reach this profit.
+Hence, keep in mind that your performance is a mix of the different elements of the strategy, your configuration, and the crypto-currency pairs you have set up.
+
+### Assumptions made by backtesting
+
+Since backtesting lacks some detailed information about what happens within a candle, it needs to take a few assumptions:
+
+- Buys happen at open-price
+- Low happens before high for stoploss, protecting capital first.
+- ROI sells are compared to high - but the ROI value is used (e.g. ROI = 2%, high=5% - so the sell will be at 2%)
+- Stoploss sells happen exactly at stoploss price, even if low was lower
+- Trailing stoploss
+  - High happens first - adjusting stoploss
+  - Low uses the adjusted stoploss (so sells with large high-low difference are backtested correctly)
+- Sell-reason does not explain if a trade was positive or negative, just what triggered the sell (this can look odd if negative ROI values are used)
```

Comment:
Should we allow it (negative roi)? -- not for this PR though...
Moreover, should we describe this here?

Reply:
it is technically allowed now - and it allows to set a "timer" on trades, forcing a sell after X minutes.

thats not how it's intendet to be used - but limiting ROI to positive values would increase the config effort (and code needed) to allow such a case:
you'd need the same number of minutes in ROI (for positive trade) and stoploss (for negative trades) ... and honestly, i don't want to add even more functionality into stoploss - it's already way too complicated with all the different options.

Reply:
I mean, should we even mention in the docs that this can be negative? To avoid questions it may cause...

Reply:
since it's allowed, and obviously people try to use it, i would do this, yes.

it maybe should be documented clearer (not in this PR ...) - as to why this configuration may make sense (limit max duration of a trade)...